contributors/guide/non-code-contributions.md

```diff
@@ -4,7 +4,7 @@
 
 *If you are interested in helping define and structure this work, [check the weekly meeting notes](https://docs.google.com/document/d/1gdFWfkrapQclZ4-z4Lx2JwqKsJjXXUOVoLhBzZiZgSk/edit#) for information on how to get involved. You can also find us in the SIG Contributor Experience [Slack channel](https://kubernetes.slack.com/messages/sig-contribex).*
 
-*All contributors welcome, new and old!*
```

Comment:
Is the original bad grammar? Is this necessary?

Reply:
The original is fine.

Reply:
Agreed.

@ramyakailas I'll close this PR because the original is also fine. I see that this is your first PR -- if you are interested to contribute, I'd recommend checking out the `good first issue`s in this repo. They are excellent issues to get started with and also get some mentorship in the process. :)

/close